src/methods/wallet-get-capabilities.ts

```diff
@@ -37,7 +41,8 @@ export async function walletGetCapabilities(
   validateParams(req.params, GetCapabilitiesStruct);
 
   const address = req.params[0];
-  const capabilities = await getCapabilities(address, req);
+  const chainIds = req.params[1];
+  const capabilities = await getCapabilities(address, chainIds, req);
```

Comment:
does it matter if chainIds is an empty array? I assume either way this case will be handled by the `getCapabilities` hook

Reply:
That should be fine as the client would just treat it as empty. Academic for the time being as we don't support any capabilities.